llvm/test/CodeGen/AArch64/hardened-jump-table-br.ll

```diff
@@ -0,0 +1,53 @@
+; RUN: llc -verify-machineinstrs -o - %s -mtriple=arm64-apple-ios -aarch64-min-jump-table-entries=1 -aarch64-enable-atomic-cfg-tidy=0 | FileCheck %s
+; RUN: llc -verify-machineinstrs -o - %s -mtriple=arm64-apple-ios -aarch64-min-jump-table-entries=1 -aarch64-enable-atomic-cfg-tidy=0 -code-model=large | FileCheck %s
```

Comment:
Nit: it would be nice to test large offsets as well, but I'm not sure how we can force such offsets artificially. So, I'm OK with current test, but large offsets tests are "nice to have" if you know a good way to implement them.